darenju/react-flip-page
# Fix npm packaging

- Remove the ```.npmignore``` as it start to get to verbose
- Add inside ```package.json``` the section ```files``` which specifiy the only directory needed, beside the already default files NPM requires (like package.json, README.md, CHANGELOG.md, ...

---

To test this, run the following commands:

(ensure to not have unversioned files in your local directory first):

```
git clean -fd .
npm install
npm run build
npm pack
```

Then, take a look at the package generated.

---

**PS.:** Sorry, I forgot to run ```npm pack``` before sending the PR, my bad.
